packages/frontend/src/blockchain/hooks/useBid.ts

```diff
@@ -0,0 +1,30 @@
+import { useChainId, useReadContract } from 'wagmi'
+import { AUCTION_ABI } from '@/blockchain/abi/auction'
+import { AUCTION_ADDRESSES } from '@/blockchain/auctionAddresses'
+import { Hex } from 'viem'
+
+export const useBid = (address: Hex | undefined) => {
+  const chainId = useChainId()
+
+  const { data: bid } = useReadContract({
+    chainId,
+    abi: AUCTION_ABI,
+    address: AUCTION_ADDRESSES[chainId],
+    functionName: 'getBid',
+    args: [address as Hex],
+    query: {
+      enabled: !!address,
+    },
+  })
+
+  return {
+    bid: bid &&
+      address && {
+        address,
+        amount: bid.amount,
+        bidderId: bid.bidderID,
+        place: Number(bid.raffleParticipantIndex),
```

Comment:
We can't set the place to raffleParticipantIndex, as it's not the place when the auction is still in the process